2022-05-11 04:37:07,962 INFO    MainThread:13032 [internal.py:wandb_internal():87] W&B internal server running at pid: 13032, started at: 2022-05-11 04:37:07.962312
2022-05-11 04:37:07,962 INFO    WriterThread:13032 [datastore.py:open_for_write():77] open: G:\Samei\Repo\RSNA-Bone-Age\wandb\offline-run-20220511_043706-1xpvyr2x\run-1xpvyr2x.wandb
2022-05-11 04:37:07,962 DEBUG   HandlerThread:13032 [handler.py:handle_request():130] handle_request: run_start
2022-05-11 04:37:07,993 DEBUG   HandlerThread:13032 [meta.py:__init__():40] meta init
2022-05-11 04:37:07,993 DEBUG   HandlerThread:13032 [meta.py:__init__():54] meta init done
2022-05-11 04:37:07,993 DEBUG   HandlerThread:13032 [meta.py:probe():214] probe
2022-05-11 04:37:08,008 DEBUG   HandlerThread:13032 [meta.py:_setup_git():204] setup git
2022-05-11 04:37:08,086 DEBUG   HandlerThread:13032 [meta.py:_setup_git():211] setup git done
2022-05-11 04:37:08,086 DEBUG   HandlerThread:13032 [meta.py:_save_pip():58] save pip
2022-05-11 04:37:08,086 DEBUG   HandlerThread:13032 [meta.py:_save_pip():72] save pip done
2022-05-11 04:37:08,086 DEBUG   HandlerThread:13032 [meta.py:_save_conda():79] save conda
2022-05-11 04:37:09,680 DEBUG   HandlerThread:13032 [meta.py:_save_conda():89] save conda done
2022-05-11 04:37:09,680 DEBUG   HandlerThread:13032 [meta.py:probe():252] probe done
2022-05-11 04:37:09,789 INFO    HandlerThread:13032 [handler.py:handle_tbrecord():640] handling tbrecord: tbrecord {
  log_dir: "tensorboard\\20220511_043706_MobileNetV2_Pre2"
  save: true
}

2022-05-11 04:37:09,789 DEBUG   HandlerThread:13032 [config_util.py:dict_from_config_file():102] no default config file found in config-defaults.yaml
2022-05-11 04:37:09,821 DEBUG   Thread-12 :13032 [tb_watcher.py:_process_events():269] Encountered tensorboard directory watcher error: [WinError 1314] A required privilege is not held by the client: 'G:\\Samei\\Repo\\RSNA-Bone-Age\\tensorboard\\20220511_043706_MobileNetV2_Pre2\\events.out.tfevents.1652269029.DESKTOP-ISIIVT7.11628.0' -> 'G:\\Samei\\Repo\\RSNA-Bone-Age\\wandb\\offline-run-20220511_043706-1xpvyr2x\\files\\events.out.tfevents.1652269029.DESKTOP-ISIIVT7.11628.0'
2022-05-11 07:13:02,008 WARNING MainThread:13032 [internal.py:wandb_internal():146] Internal process interrupt: 1
2022-05-11 07:13:03,664 WARNING MainThread:13032 [internal.py:wandb_internal():146] Internal process interrupt: 2
2022-05-11 07:13:03,664 ERROR   MainThread:13032 [internal.py:wandb_internal():149] Internal process interrupted.
2022-05-11 07:13:03,680 INFO    SenderThread:13032 [sender.py:finish():1070] shutting down sender
2022-05-11 07:13:03,961 INFO    WriterThread:13032 [datastore.py:close():281] close: G:\Samei\Repo\RSNA-Bone-Age\wandb\offline-run-20220511_043706-1xpvyr2x\run-1xpvyr2x.wandb
2022-05-11 07:13:03,961 INFO    HandlerThread:13032 [handler.py:finish():731] shutting down handler
2022-05-11 07:13:10,430 INFO    MainThread:13032 [internal.py:handle_exit():77] Internal process exited
